client/src/services/notificationManager.ts

```diff
@@ -131,28 +133,33 @@ export class NotificationManager {
     }
   }
 
-  async addToList(notification: Notification): Promise<void> {
-    this.notifications.push(notification);
+  addToList(notification: Notification): void {
+    this.notifications.value.unshift(notification);
   }
 
   markAsRead(id: string, read = true): void {
-    const notification = this.notifications.find((n) => n.id === id);
+    const notification = this.notifications.value.find((n) => n.id === id);
 
     if (notification) {
       notification.read = read;
     }
+    this.notifications.value = [...this.notifications.value];
```

Comment:
What does this do?

Reply:
I think that's it was a test, probably useless now, will delete

Reply:
deleted